03_letters/anishaNair/main_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := `a : 2
+b : 1
+`
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+func TestCountLettersOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// Then
+	expected := map[rune]int{104: 1, 97: 2, 110: 1, 105: 1, 115: 1}
```

Comment:
easier to read as: `expected := map[rune]int{'n': 1, 'a': 2, 's': 1, 'i': 1, 'h': 1}`